react/ui-components/src/atoms/SelectionTag.js

```
@@ -1,22 +1,61 @@
   allowMultipleSelection = true,
   selected,
-  withContainer
+  withContainer,
+  populators = {},
 }) => {
-  const { t } = useTranslation();
+  const { t: i18nT } = useTranslation();
+  const t = populators?.t || i18nT;
   const [selectedOptions, setSelectedOptions] = useState(selected || []);
 
+  const { isLoading, data } = window?.Digit?.Hooks.useCustomMDMS(
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)